old-packages/Core/src/Collections/List/.candy

```diff
@@ -8,61 +8,14 @@ use ..Iterate
 public use .Array
 
 public trait List[Item]: Iterate[Item] {
-  public static fun filled(length: Int /* UInt */, item: Item): List[Item] {
-    Array[Item].filled(length, item)
-  }
   public static fun generate(length: Int, generator: (Int) -> Item): List[Item] {
     Array[Item].generate(length, generator)
   }
-  public static fun empty(): List[Item] { Array[Item].empty() }
```

Comment:
Just to make sure: We decided to not have a `list.empty` in favor of the literal, right?

Reply:
Correct